docs/start/quickstart.md

```diff
@@ -2,73 +2,56 @@
 title: Quickstart
 position: 2
 ---
-# Quickstart
-
-Infra is rolling out it's Software as a Service offering which makes it easier than ever to setup infrastructure access for your organization. This page will show you everything you need to do to get started.
 
-As you scroll down, you will see a lot of text and might think that this is going to be hard. But really there are 5 steps as shown in the Table of Contents to the right. Everything else is just describing what to do just in case you run into any issues.
+# Quickstart
 
-## Signup to Create a New Organization
+## 1) Signup to Create a New Organization
```

Comment:
Do we want to use `1.` (preferred) across the docs or `1)`?

Reply:
yup